src/components/MoneyRequestConfirmationList.js

```diff
@@ -79,6 +79,12 @@ const propTypes = {
 
     /** The reportID of the request */
     reportID: PropTypes.string,
+
+    /** File path of the receipt */
+    receiptPath: PropTypes.string,
```

Comment:
A bit lost here. What's the difference between the path and the source?

Reply:
File path is the location of the image on your computer or phone.

File source is the file name of the receipt.